products/error_tracking/frontend/scenes/ErrorTrackingFingerprintsScene/errorTrackingIssueFingerprintsSceneLogic.tsx

```diff
@@ -33,6 +32,11 @@ export const errorTrackingIssueFingerprintsSceneLogic = kea<errorTrackingIssueFi
         loadIssue: true,
         split: (exclusive: boolean) => ({ exclusive }),
         setSelectedFingerprints: (fingerprints: string[]) => ({ fingerprints }),
+        loadFingerprintSamples: (issue: ErrorTrackingRelationalIssue, fingerprints: ErrorTrackingFingerprint[]) => ({
+            issueId: issue.id,
+            firstSeen: issue.first_seen,
+            fingerprints: fingerprints ? fingerprints.map((fingerprint) => fingerprint.fingerprint) : null,
```

Comment:
Don't know if it's strictly "bad" but you don't often see manipulation of the action inputs. Will fingerprints ever be falsey based on the `fingerprints: ErrorTrackingFingerprint[]` type? Maybe we should just pass it through and deal with the empty array elsewhere